ExecCpp/UnitTests/Shu-Osher/prob.H

```diff
@@ -0,0 +1,157 @@
+#ifndef _PROB_H_
+#define _PROB_H_
+
+#include <AMReX_Print.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_Geometry.H>
+#include <AMReX_FArrayBox.H>
+
+#include "mechanism.h"
+
+#include "PeleC.H"
+#include "IndexDefines.H"
+#include "Constants.H"
+#include "EOS.H"
+#include "Tagging.H"
+#include "Transport.H"
+#include "ProblemDerive.H"
+#include "prob_parm.H"
+
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+pc_initdata(
+  int i,
+  int j,
+  int k,
+  amrex::Array4<amrex::Real> const& state,
+  amrex::GeometryData const& geomdata)
```

Comment:
This wont compile with development because of changes to the way we treat problem parameters